driver-core/build.gradle

```diff
@@ -17,6 +17,7 @@
 
 plugins {
     id 'com.github.gmazzo.buildconfig' version '3.0.3'
+    id('conventions.testArtifacts')
```

Comment:
Configures `driver-core` to produce test artifacts that can be consumed by other projects (eg the Scala project).